src/API/OnboardingTasks.php

```diff
@@ -187,12 +187,12 @@ public static function sanitize_special_column_name_regex( $value ) {
 	private static function get_homepage_cover_block( $image ) {
 		if ( ! empty( $image['url'] ) && ! empty( $image['id'] ) ) {
 			return '<!-- wp:cover {"url":"' . esc_url( $image['url'] ) . '","id":' . intval( $image['id'] ) . '} -->
-			<div class="wp-block-cover has-background-dim" style="background-image:url(' . esc_url( $image['url'] ) . ')"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"' . __( 'Write title…', 'woocommerce-admin' ) . '","fontSize":"large"} -->
+			<div class="wp-block-cover has-background-dim" style="background-image:url(' . esc_url( $image['url'] ) . ')"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"' . __( 'Write title…', 'woocommerce-admin' ) . '","textColor":"white","fontSize":"large"} -->
 			<p class="has-text-align-center has-large-font-size">' . __( 'Welcome to the store', 'woocommerce-admin' ) . '</p>
 			<!-- /wp:paragraph -->
 
-			<!-- wp:paragraph {"align":"center"} -->
-			<p class="has-text-align-center">' . __( 'Write a short welcome message here', 'woocommerce-admin' ) . '</p>
+			<!-- wp:paragraph {"align":"center","textColor":"white"} -->
+			<p class="has-text-color has-text-align-center">' . __( 'Write a short welcome message here', 'woocommerce-admin' ) . '</p>
```

Comment:
Why is `has-text-color` needed? The CSS rules are a bit complicated around this class being/not being present.

Reply:
I wondered the exact same thing - this template, from what I can tell, is pretty much a copy/paste from the block editor, so I was following that pattern in this part of the PR. When I applied the white font color from the block editor sidebar, the classname was added.

I'd have to go dig into gutenberg to see the original reason for `has-text-color`

Reply:
That's cool - there's probably a great reason for it